### DISCUSSION TITLE
[Feature request]:In TOPAS-nbio, are these chemical materials provided, such as EDTA, TRIS, and WR-1065?

### DISCUSSION THREAD

**Discussion by v587aaaaa3** [topas-nbio/TOPAS-nBio#48] [Feature-Request]
### Feature request

I couldn't seem to find a definition of "Compounds" in TOPAS and TOPAS-nbio. If I want to use materials like the ones mentioned above, is there a relevant material database available?

### TOPAS-nBio category

Geometry

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by tiagoaaazevedo1** [comment]
Hi! TOPAS has access to the predefined materials of GEANT4. Here you can find a list of some of NIST compounds:

https://geant4-userdoc.web.cern.ch/UsersGuides/ForApplicationDeveloper/html/Appendix/materialNames.html

However, it seems that they don't have the chemical materials that you are looking for, so maybe it's better to define them using the method shown in the TOPAS docs:

https://topas.readthedocs.io/en/latest/parameters/material.html

I hope this helps! Let me know if you have any more questions. :)
Best regards,

T. Azevedo
